app/components/publications/publication/translations/translation-requested-panel.hbs

```diff
@@ -0,0 +1,80 @@
+<div class="vlc-procedure-step vlc-procedure-step--nopadding">
+  <div class="vlc-procedure-step__badge-wrapper">
+    <Auk::Badge @icon="mail" @skin="primary" @size="medium" class="vlc-procedure-step__badge"/>
+  </div>
+  <Auk::AccordionPanel @isDefaultActive={{true}} as |accordionPanel|>
+    <accordionPanel.Header
+      @size="large"
+      @title={{t "requested-translation"}}
+      @subtitle={{moment-format @requestActivity.startDate  "DD.MM.YYYY"}}
+    as |header|>
+      <header.Actions as |actions|>
+        <actions.Item>
+          <DropdownMenu
+            @skin="borderless"
+            @icon="three-dots"
+            as |Menu|>
+            <Menu.Item
+              @skin="danger"
+              {{on "click" this.promptDeleteRequest}}
+            >
+              {{t "delete-translation-request"}}
+            </Menu.Item>
+          </DropdownMenu>
+        </actions.Item>
+      </header.Actions>
+    </accordionPanel.Header>
+    <accordionPanel.Body>
+      {{#each @requestActivity.usedPieces as |piece|}}
+        <div class="auk-o-flex auk-o-flex--horizontal auk-o-flex--spaced-wide">
+          <Auk::List::Item class="auk-o-flex auk-o-flex--horizontal auk-o-flex--spaced-wide">
+            <div class="auk-o-flex auk-o-flex--horizontal auk-o-flex--spaced">
+              <Auk::Badge @icon="paperclip" @size="regular" />
+              <h5 class="auk-h4 auk-u-m-0">{{piece.file.downloadFilename}}</h5>
+            </div>
+            <Auk::ButtonLink
+              @layout="icon-only"
+              @icon="download"
+              @href={{piece.file.namedDownloadLink}}
+              download={{piece.file.downloadFilename}}
+            >
+              {{t "download"}}
+            </Auk::ButtonLink>
+            <Auk::ButtonLink
+              @layout="icon-only"
+              @icon="book"
+              @href={{piece.viewDocumentURL}}
+              target="_blank"
+            >
+              {{t "read"}}
+            </Auk::ButtonLink>
+          </Auk::List::Item>
+        </div>
+      {{/each}}
+      {{#if @requestActivity.email}}
+        <div class="auk-hr"></div>
+        <div class="auk-content">
+          <h4 class="auk-u-m-0">{{t "subject"}}</h4>
+          <p>{{@requestActivity.email.subject}}</p>
+          <h4 class="auk-u-m-0">{{t "message"}}</h4>
+          {{! TODO: Remove inline style }}
+          {{! template-lint-disable no-inline-styles }}
+          {{! p-tag opening and closing on 1 line to prevent whitespace to be rendered. }}
+          <p style="white-space:pre-line;">
+            {{@requestActivity.email.message}}
+          </p>
```

Comment:
Ofwel moet de comment weg, ofwel moet de paragraph weer op 1 lijn.

Reply:
Ik zie in huidige code nog plaatsen waar diezelfde comment stond en ook linter op gerund is
(proofs/requests/template, translations/requests/template)

Gebruik `{{! prettier-ignore }}` zodat prettier die ene regel niet formateerd bij autoformat.